commonmark/pom.xml

```diff
@@ -8,6 +8,7 @@
     </parent>
 
     <artifactId>commonmark</artifactId>
+    <packaging>bundle</packaging>
```

Comment:
I'm a bit uncomfortable with changing the packaging type here. Is there a way to add the manifest entries without changing this?

Reply:
Actually there is, but adds more configuration. However there are no issues with bundle packaging. The plugin exists for a lot of time and works well with all other plugins.

[1] https://bnd.bndtools.org/tools/felix-maven.html